### ISSUE TITLE
[IDEA] Improve UI accessibility

### ISSUE THREAD

**Issue by Leilei332** [Jermolene/TiddlyWiki5#9169]
**Is your feature request related to a problem? Please describe.**
There are many a11y problems in tiddlywiki. Like the screen reader describes Theme, Layout and Storyview choosers as "links". Tabs are described as "switch buttons".

**Describe the solution you'd like**
We should add `listbox` ,`option` role and `aria-selected` attribute to the choosers, and use `tab`, `tablist` ,`tabpanel` role and `aria-orientation` role for the tabs macro.

TOC and tree macro should use `tree`, `treeitem`, `group` role and `aria-expanded` attribute.

Alerts should have `alert` role, while modals should have `dialog` role with `aria-modal` set to true.

**Additional context**
* See https://www.w3.org/WAI/ARIA/apg/patterns/
* #9167
* #9154

**Comment by Jermolene** [comment]
Hi @Leilei332 I would love to see those accessibility improvements.
